docs/concepts/best-practices/function-naming.rst

```diff
@@ -1,51 +0,0 @@
-===============
```

Comment:
I feel like a bunch of this content was not incorporated in the new page?

Reply:
I should actually revert deletion. 

The goal would be to remove the `best-practices` section and have all the info under either `Concepts` or `How to`.

What wasn't included in the `Node` page would probably fit best under a `Hamilton for teams` page under `How to`